lib/OpenQA/WebAPI/Controller/File.pm

```diff
@@ -126,9 +126,9 @@ sub download_asset {
     my $path = $self->param('assetpath');
     return $self->reply->not_found if $path =~ qr/\.\./;
 
-    my $static = Mojolicious::Static->new;
-    $static->paths([$OpenQA::Utils::assetdir]);
-    return $self->rendered if $static->serve($self, $path);
+    my $file = path($OpenQA::Utils::assetdir, $path)->to_string;
+    return $self->reply->not_found unless -f $file && -r _;
```

Comment:
What does `-r _`? I think that deserves a comment.

Reply:
That's actually a commonly used Perl optimization. The `_` is a placeholder that makes `-r` reuse the result of the previous `stat` call done by the `-f`.

Reply:
ah, ok